test/e2e/gameserver_test.go

```diff
@@ -109,12 +110,13 @@ func TestSDKSetAnnotation(t *testing.T) {
 		if err != nil {
 			return true, err
 		}
-		return gs.ObjectMeta.Annotations != nil, nil
+
+		_, ok := gs.ObjectMeta.Annotations[annotation]
+		return ok, nil
 	})
 
 	assert.Nil(t, err)
-	assert.NotEmpty(t, gs.ObjectMeta.Annotations["stable.agones.dev/sdk-timestamp"])
-
+	assert.NotEmpty(t, gs.ObjectMeta.Annotations[annotation])
```

Comment:
I distinctly remember seeing failures where "sdk-timestamp" was set but version was not.

Reply:
Can you try:

```go test -count=100 -timeout 90s agones.dev/agones/test/e2e -run TestSDKSetAnnotation```

Reply:
That would be strange, as the version annotation is set via a mutation webhook.

Ran this 50 times (in batches to fit under the timeout - also added a defer to delete the gameserver), no errors :man_shrugging: 

(Nice trick with `-count=100` didn't know that one, will steal it :+1: )